.header.md

```diff
@@ -14,11 +14,7 @@ This module creates and configures an AWS Transfer Server with the following fea
 
 ```hcl
 module "transfer_sftp" {
-<<<<<<< HEAD
-  source = "github.com/aws-ia/terraform-aws-transfer-family//modules/transfer-server"
-=======
-  source = "aws-ia/aws-transfer-family-terraform-modules/aws//modules/transfer-server"
->>>>>>> d7156a9bffd0c51d4fc106f61f039b72f347ca84
+  source = "github.com/aws-ia/terraform-aws-transfer-family/tree/main/modules/transfer-server"
```

Comment:
the source needs to be this : source = "aws-ia/aws-transfer-family-terraform-modules/aws//modules/transfer-server". 

Not sure if the github url works.